cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestCommandTest.java

```diff
@@ -72,6 +72,18 @@ public void testTestBalFile() throws IOException {
         testCommand.execute();
     }
 
+    @Test(description = "Test a valid ballerina file with periods in the file name")
+    public void testTestBalFileWithPeriods() throws IOException {
```

Comment:
Cant we remove this `IOException`, seems like it's never thrown.

Reply:
Removed `throws IOException`